lib/lita/handlers/github_pinger.rb

```diff
@@ -116,6 +120,49 @@ def act_on_assign(body, response)
         response
       end
 
+      def act_on_pr_opened(body, response)
+        puts "Detected that someone opened a #{type.tr('_', ' ')}."
+        if type.nil?
+          puts 'Neither pull request or issue detected, exiting...'
+          return
+        end
+
+        if config.enable_round_robin
+          puts "round robin is enabled, selecting the next engineer.."
+          redis_key = 'lita-github-pinger:nextreviewer'
```

Comment:
maybe throw this in as a `freeze` at the top level of the module since we're never going to modify this value while this function is executing.